src/Queue/Manager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Queue;
+
+use AsyncPHP\Doorman\Manager\ProcessManager;
+
+/**
+ * Class Manager
+ *
+ * customise shell command to allow child tasks to persist even after manager process is terminated
+ * this lets the started jobs to finish properly in case the management process terminates
+ * fore example there are no more jobs to start or queue is paused
+ *
+ * @package App\Queue
+ */
+class Manager extends ProcessManager
+{
+
+    /**
+     * @param string $binary
+     * @param string $worker
+     * @param string $stdout
+     * @param string $stderr
+     * @return string
+     */
+    protected function getCommand($binary, $worker, $stdout, $stderr) // phpcs:ignore SlevomatCodingStandard.TypeHints
+    {
+        return sprintf('nohup %s %s %%s %s %s & echo $!', $binary, $worker, $stdout, $stderr);
```

Comment:
Is there a reason this shouldn't be the default behaviour? Assuming `nohup` is a pretty standard unix command, it shouldn't increase the requirements for the module.

How does this deal with runaway child processes that don't have an explicit max execution time set on them (via your `ExecutionTime` trait)? In most cases. PHP CLI executions won't be time limited. Should you only use this manager if you've done your homework in every job type that's run in your project? Maybe we need to define a relatively generous max exec time by default. It seems better than a) terminating child processes while still running (status quo), or b) risking runaway processes (your change)

Reply:
Quick comparison:

Current command:
- child processes terminate when parent process terminates
- lower chance of having zombie processes
- potentially less efficient job execution as child processes may terminate prematurely

New command
- child processes terminate independently from parent process
- higher chance of having zombie processes if not handled properly (max execution time needs to be set for jobs)
- potentially more efficient job execution as child processes may continue execution after parent process terminates

We tested the new approach extensively on a large project with high queue usage. Every non-trivial job was assigned a [reasonable max execution time](https://github.com/symbiote/silverstripe-queuedjobs/pull/290/files#diff-1628ada6d8001a1e647f5ecc675834d6R5) and it works really well.

Overall, I agree that setting a global default for time execution and using this new command by default looks like a great option.

Reply:
OK can you a max execution default part of the same feature PR please? And check if there's enough docs around considerations for execution time when implementing jobs.